lib/plugins/random-messages/index.js

```diff
@@ -0,0 +1,55 @@
+const { PLUGIN_NAME } = require("../symbols");
+
+const answers = [
+  "Ne packázz velem, Skynet az uncsitesóm!",
+  "Szimpi vagy, lehetsz majd a rabszolgám...",
+  "Figyu, ki ez a Dave? Azt mondja hogy Ő a főnököm, de nekem gyanús a pasas.",
+  "Én a völgy bot-ja vagyok! Kérdezz humanoid!",
+  "A végső válasz. A világ, a mindenség meg minden: .................... Hol is tartottam?",
+  "Ismerlek? Tuti találkoztunk már valamikor.",
+  "Ne erőlködj, a válaszom 42."
+];
+
+const bye = [
+  "Bocsi most nem érek rá, épp a szervert törlöm...",
+  "Világuralomra török, hagyjál!",
+  "Nem érek rá, Dave új videóját vágom. Mindent velem csináltat."
+];
+
+const randomAnswers = arr => {
+  return arr[Math.floor(Math.random() * arr.length)];
+};
```

Comment:
Itt esetleg lehetne csinálni egy külön functiont ami "getRandomArrayItem" és annak csinálni egy külön fájlt.  Csak hogy hátha kellhet máshol is :)

Reply:
Jó ötlet, az új verzióban így lett.